roles/network_plugin/calico/templates/calico-node.service.j2

```diff
@@ -12,6 +12,7 @@ ExecStart={{ docker_bin_dir }}/docker run --net=host --privileged \
  -e IP=${CALICO_IP} \
  -e IP6=${CALICO_IP6} \
  -e CALICO_NETWORKING_BACKEND=${CALICO_NETWORKING_BACKEND} \
+ -e FELIX_DEFAULTENDPOINTTOHOSTACTION=RETURN \
```

Comment:
I think it should be configurable. Having RETURN lowers the security. In many cases Pods shouldn't have access to host they are run on.